plugins/node/opentelemetry-instrumentation-aws-lambda/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@opentelemetry/instrumentation-aws-lambda",
   "version": "0.41.0",
-  "description": "OpenTelemetry AWS Lambda automatic instrumentation package.",
+  "description": "OpenTelemetry instrumentation for the execution of AWS Lambda runtime handler",
```

Comment:
THis description is still a little cryptic to me. Would something like this work?

`OpenTelemetry instrumentation for AWS Lambda function invocations`

Reply:
like it. done 👍